pyrokinetics/local_geometry/LocalGeometry.py

```diff
@@ -26,14 +50,159 @@ def __init__(self, *args, **kwargs):
             super(LocalGeometry, self).__init__(_data_dict)
 
     # TODO replace this with an abstract classmethod
-    @not_implemented
-    def load_from_eq(self, eq, psi_n=None):
+    def load_from_eq(self, eq, psi_n=None, show_fit=False, **kwargs):
         """ "
         Loads LocalGeometry object from an Equilibrium Object
 
         """
+        R, Z = eq.get_flux_surface(psi_n=psi_n)
+
+        R_major = eq.R_major(psi_n)
+
+        rho = eq.rho(psi_n)
+
+        r_minor = rho * eq.a_minor
+
+        Zmid = (max(Z) + min(Z)) / 2
+
+        fpsi = eq.f_psi(psi_n)
+        B0 = fpsi / R_major
+
+        drho_dpsi = eq.rho.derivative()(psi_n)
+
+        pressure = eq.pressure(psi_n)
+        q = eq.q(psi_n)
+
+        dp_dpsi = eq.q.derivative()(psi_n)
+
+        shat = rho / q * dp_dpsi / drho_dpsi
+
+        dpressure_drho = eq.p_prime(psi_n) / drho_dpsi
+
+        beta_prime = 8 * pi * 1e-7 * dpressure_drho / B0**2
+
+        b_poloidal = eq.get_b_poloidal(R, Z)
+
+        # Store Equilibrium values
+        self.psi_n = psi_n
+        self.rho = float(rho)
+        self.r_minor = float(r_minor)
+        self.Rmaj = float(R_major / eq.a_minor)
+        self.Z0 = float(Zmid / eq.a_minor)
+        self.a_minor = float(eq.a_minor)
+        self.f_psi = float(fpsi)
+        self.B0 = float(B0)
+        self.q = float(q)
+        self.shat = shat
+        self.beta_prime = beta_prime
+        self.pressure = pressure
+        self.dpressure_drho = dpressure_drho
+
+        self.R = R
+        self.Z = Z
+        self.b_poloidal = b_poloidal
+
+        # Calculate shaping coefficients
+        self.get_shape_coefficients(self.R, self.Z, self.b_poloidal)
+
+        # Bunit for GACODE codes
+        self.bunit_over_b0 = self.get_bunit_over_b0()
+
+        if show_fit:
+            self.plot_fits()
+
+    def load_from_lg(self, lg, verbose=False, show_fit=False):
```

Comment:
Is this method on the right class? It looks like a copy method rather than a constructor, but says it creates a `FourierCGYRO`

Also, let's expand those abbreviations :stuck_out_tongue: 

```suggestion
    def load_from_local_geometry(self, local_geometry, verbose=False, show_fit=False):
```